Enable copy/paste of commands (#669)
rkvst/rkvst-docs: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md

    ```

1. Save the base64 encoded `JWT` using `jq` to find `.access_token` (See [./jq](https://jqlang.github.io/jq/) for more info)

    ```bash
    TOKEN=$(echo -n $RESPONSE | jq -r .access_token)
    echo $TOKEN
    ```

1. Create a Bearer Token file with `0600` permissions for reference by `curl` commands

    ```bash
    echo Authorization: Bearer $TOKEN > rkvst-bearer.txt
    cat rkvst-bearer.txt
    ```

## Testing Token Creation

You can test access to the RKVST API using any of the standard API calls. GETing `Assets` is a simple test.

If successful, you will see a list of the assets the Integration has access to in the tenancy.
{{< note >}}
Note this may be an empty response if no assets are being shared with the user; this is an expected secure by default behavior.
{{< /note >}}

Otherwise, check the [Assets OpenAPI Reference](../../api-reference/assets-api/#assets-openapi-reference) for more detailed information on the response codes you may expect if authentication fails and what they mean.

### View Existing Assets

To test the creation of the Custom integration and the configuration of the bearer token file (`rkvst-bearer.txt`), query the assets API

```bash

> curl -H "@rkvst-bearer.txt" \

Reviewer: might be an idea to explicitly add the GET as to make it clear and it aligns with the current API documentation

@SteveLasker

Reviewer 2: Good spot, people who are new to curl or REST might need the examples to be free of assumptions.

Author: Thanks @mmwilder26, it was the warning from the `curl` command that GET is the default that caused me to remove it. 
I do like the explicit command.

Author: I just realized, the text: 
> Unnecessary use of -X or --request, GET is already inferred.

Is only present when `-v` is also passed.
```bash
curl -v -X GET -H "@rkvst-bearer.txt"     https://app.rkvst.io/archivist/v2/assets | jq
Note: Unnecessary use of -X or --request, GET is already inferred.
```

I've updated to include `-X GET` and remove `-v` as it was overly noisy for the getting started experience.

  - /docs/rkvst-basics/getting-access-tokens-using-app-registrations/
---

Non-interactive access to the RKVST platform is managed by creating `Integrations` with either a Custom Integration or one of the built-in Integrations. This is done using either the `Settings` Menu in the RKVST UI or by using the App Registrations API directly.
{{< note >}}
**Note:** App Registration is the previous name for an Integration.
{{< /note >}}
 
`Integrations` have a `CLIENT_ID` and `SECRET` used to authenticate with RKVST IAM endpoints using [JSON Web Tokens](https://jwt.io/introduction/) (JWT).

RKVST authentication uses the industry-standard OIDC Client Credentials Flow.

## Creating a Custom Integration

{{< note >}}
**Note:** Creating App Registrations requires **Administrator** privileges.  
If `Settings` does not appear in the navigation, see your RKVST Administrator for access.
{{< /note >}}

### Using the RKVST App to Create an Integration (First-Time Setup)

> 

Reviewer 2: Is there a reason why these bullet points are all 1. ? I still see a numbered sequence in the browser.

Reviewer 2: I've just found the reason while reviewing the style guide changes :-)

1. As an Administrator, open the <a href="https://app.rkvst.io/" target="_blank">RKVST App</a>
1. Navigate to `Settings` on the sidebar
1. Navigate to the `Integrations` tab
  {{< img src="IntegrationsTab.png" alt="Rectangle" caption="<em>Navigate to Settings, then Integration</em>" class="border-0" >}}
1. Click the `Custom` box to create a Custom Integration
1. Enter any `Display Name` you'd like
  {{< img src="Confirm.png" alt="Rectangle" caption="<em>Completed Web Registration</em>" class="border-0" >}}
  {{< note >}}
  Optionally add any `Custom claims` at this step by clicking the `+ Add` button. Ensure the `Name` _does not start_ with `jit_` (RKVST reserved names) or use any other [well-known reserved claims](https://auth0.com/docs/security/tokens/json-web-tokens/json-web-token-claims#reserved-claims).
  {{< /note >}}  
1. Once complete, click `Confirm` to complete the custom integration
1. You will then be presented with the `CLIENT_ID` and `SECRET` required by the archivist token endpoint

>     {{< caution >}}

Reviewer 2: ```suggestion
{{< caution >}}
```

Reviewer 2: These spaces prevent the Caution box from displaying correctly.

Author: @pcnorth, curious why you're seeing this differently. Here's what I see locally:
![image](https://github.com/rkvst/rkvst-docs/assets/7647382/c34f2f7b-b145-43b1-aaf5-ea9c5b5e55a2)

What's also interesting: if I undent the block from lines 49-52, the ordered list still maintains the number counting. 🤔

>     **Caution:** Save the `CLIENT_ID` and `SECRET` to a password manager or secret management service as the `SECRET` can **not** be viewed again. A new `SECRET` can be regenerated by editing the Integration, invalidating the previous `SECRET`.

Reviewer 2: ```suggestion
**Caution:** Save the `CLIENT_ID` and `SECRET` to a password manager or secret management service as the `SECRET` can **not** be viewed again. A new `SECRET` can be regenerated by editing the Integration, invalidating the previous `SECRET`.
```

Author: see above

>     {{< /caution >}}  

Reviewer 2: ```suggestion
{{< /caution >}}  
```

Author: see above

>     {{< img src="RecordClientIDandSecret.png" alt="Rectangle" caption="<em>Record your Client ID and Secret</em>" class="border-0" >}}

Reviewer 2: ```suggestion
{{< img src="RecordClientIDandSecret.png" alt="Rectangle" caption="<em>Record your Client ID and Secret</em>" class="border-0" >}}
```

Author: see above

2. Copy the `CLIENT_ID` and `SECRET` to a password manager for future reference

> 3. Skip to [Grant Permissions to Custom Integration](#grant-permissions-to-custom-integration) to configure an Access Policy

Reviewer 2: Jon suggested deleting the section "Using the App Registrations API to Create an Integration". When this is done there is no need to skip to the section.
```suggestion
9. The next section describes how to configure an Access Policy for the Integration
```

> ### Using the App Registrations API to Create an Integration

Reviewer 2: I agree with Jon that this section should be deleted. I've never used it and the article flows much better without the interruption.
